apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -0,0 +1,487 @@
+import * as React from 'react';
+import {
+  usePopper,
+  createArrowStyles,
+  PositioningProps,
+  PopperVirtualElement,
+  PopperRefHandle,
+} from '@fluentui/react-positioning';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { useMergedRefs } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+import { storiesOf } from '@storybook/react';
+import { useFluent } from '@fluentui/react-shared-contexts';
+
+const useStyles = makeStyles({
+  wrapper: {
+    display: 'flex',
+    ...shorthands.gap('5px'),
+    backgroundColor: tokens.colorNeutralBackground1,
+
+    '& .target': {
+      color: tokens.colorNeutralForeground1,
+      backgroundColor: tokens.colorNeutralBackground1,
+      ...shorthands.border('2px', 'dashed', 'green'),
+      width: '400px',
+      height: '200px',
+    },
+  },
+
+  gridWrapper: {
+    width: '400px',
+    height: '400px',
+    display: 'grid',
+    gridTemplateColumns: 'auto auto auto',
+    columnGap: '20px',
+    rowGap: '50px',
+  },
+
+  boundary: {
+    ...shorthands.border('2px', 'dashed', 'red'),
+  },
+
+  box: {
+    ...shorthands.padding('15px'),
+    ...shorthands.border('1px', 'solid', 'blue'),
+    backgroundColor: 'white',
+  },
+
+  arrow: {
+    ...createArrowStyles({ arrowHeight: 5 }),
```

Comment:
nit: I would make an arrow a bit bigger, for example "7" or "8" to make it more visible